libkbfs/folder_block_manager.go

```diff
@@ -1029,7 +1037,7 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 	ctx, cancel := context.WithCancel(fbm.ctxWithFBMID(context.Background()))
 	fbm.setReclamationCancel(cancel)
 	defer fbm.cancelReclamation()
-	defer timer.Reset(fbm.config.QuotaReclamationPeriod())
+	defer timer.Reset(fbm.config.Mode().QuotaReclamationPeriod())
```

Comment:
This fires the timer in modes where `QuotaReclamationPeriod()` is 0. Just making sure you've considered it!

Reply:
Oh I meant to delete this comment but forgot. Ignore this!